hera_pspec/pspecdata.py

```diff
@@ -833,13 +869,13 @@ def set_R(self, d):
         for k in d:
             self._R[k] = d[k]
 
-    def R(self, key):
+    def R(self, key, average_times=False):
         """
         Return the data-weighting matrix R, which is a product of
         data covariance matrix (I or C^-1), diagonal flag matrix (Y) and
         diagonal tapering matrix (T):
 
-        R = sqrt(T^t) sqrt(Y^t) K sqrt(Y) sqrt(T)
+        R = T K Y
```

Comment:
did we decide on making this change? or is this not consistent with what's happening?

Reply:
This is outdated documentation. T is on the left if diagonal_taper is False but otherwise is on either side. I've updated the docstring.